docs/getting-started/timelords.md

```diff
@@ -6,10 +6,289 @@ slug: /timelords
+sudo apt-get update
+
+## Install ASIC repo
+sudo apt-get install chiavdf-hw
+
+# Launch the ASIC timelord services (if using a cluster verify the IP address is correct and launch with only 1 for each)
+hw_vdf_client --ip 127.0.0.1 8000 3
+
+## Launch the timelord-only chia service
+chia start node timelord-only
+
+```
+
+## ASIC Timelord Systemd Setup
+
+Below is an example of a systemd service file to run the ASIC hw vdf processes.
+NOTE - make sure to replace `USERNAME` with your system's username.
+
+```bash
+# Create systemd service
+sudo nano /etc/systemd/system/chiahw-vdf.service
+
+# Copy the data from below replacing USERNAME with your system's username
+```
+
+### Example ASIC systemd File
+
+```bash
+[Unit]
+Description=Chia HW VDF Service
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/hw_vdf_client --ip 127.0.0.1 8000 3
+User=USERNAME
+Group=USERNAME
+LimitNOFILE=1048576
+LimitNPROC=1048576
+LimitCORE=infinity
+
+[Install]
+WantedBy=multi-user.target
+```
+
+```bash
+# Save and start the systemd service
+CTRL+O # save the file
+CTRL-X # exit the nano editor
+
+# Reload and start the systemd services
+sudo systemctl daemon-reload
+sudo systemctl enable chiahw-vdf.service
+sudo systemctl start chiahw-vdf.service
+sudo systemctl status chiahw-vdf.service
+```
+
+### Using the systemd Service
+
+Restart the ASIC systemd software:
+`sudo systemctl restart chiahw-vdf.service`
+
+Stop the ASIC systemd software:
+`sudo systemctl stop chiahw-vdf.service`
+
+Check status of the ASIC systemd software:
+`sudo systemctl status chiahw-vdf.service`
+
+## Troubleshooting a Timelord
+
+For troubleshooting steps please refer to the documentation [here](/troubleshooting/timelords).
+
+---
+
+## Timelord support
+
+Join Our [Discord](https://discord.gg/chia) and jump into the #support channel for support
+
+---
+
+## Timelord FAQ
+
+### What are the hardware requirements for running a Timelord?
+
+There are no specific requirements as timelords are a fastest wins process. This means that those with higher end hardware are more likely to generate Proofs of Time than those with lower end hardware.
+Currently, a modern gaming PC with 8 cores and 8 GB of RAM is recommended.
+
+### Can a Singe ASIC Compete with an ASIC Cluster?
+
+The nature of timelords is to create three VDF chains, one can create the chains themselves in parallel (i.e. one on each ASIC) but one cannot break down the VDFs themselves to parallelize them.
+This means that the ASIC cluster will always have an advantage but there are times when a single ASIC can reasonably compete. This almost always requires the block farming node to be closer in physical proximity to the single ASIC than to the ASIC cluster establishing a minor time advantage for the single ASIC
+
+### Can I Overclock the ASIC to Get More Performance or Higher IPS?
```

Comment:
How about another question regarding the optimal voltage for running an ASIC?

Reply:
Question regarding voltage has eben added. Thank you for the suggestions!